src/components/Organisms/VideoCard/VideoCard.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { cleanup } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import VideoCard from './index';
+
+describe('Test VideoCard Component', () => {
+  afterEach(cleanup);
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<VideoCard />, div);
+  });
+
+  it('matches snapshot', () => {
```

Comment:
Ditto on snapshot testing.

Reply:
Some adjustments were made here to align with other changes. (Preview in Github is not working at least for me, you should see the changes directly in the file)